.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
+<!--
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>   …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the question itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+-->
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Actual behavior
+<!-- A clear and concise description of what happened instead. -->
+
+## Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+# Environment:
+<!-- (please complete the following information) -->
+
+## Java/JDK
+<!-- `java -version` -->
+
+## Build tool version
+<!-- Remove unused sections -->
+
+### Gradle
+<!-- `gradlew -version` -->
+
+### Apache Maven
+<!-- `mvn -version` -->
+
+## Operating System
+<!--  Linux, Windows, Mac etc. -->
+
+## IDE
+<!--  IntelliJ, Eclipse etc. -->
+
+## Build-tool dependencies used
+
+### Gradle/Grails
+    implementation 'org.spockframework:spock-core:2.0-groovy-3.0'
```

Comment:
Maybe add Groovy (but not `groovy-all`) here and in the Maven example as that is the Groovy version that is interesting after all?

Reply:
I mean there is a lot of dependencies that are interesting for us, e.g. junit, hamcrest, bytebuddy.  We could also suggest adding the output of 
`mvn dependency:tree -DoutputFile=dependency-report.txt` and
`./gradlew -q dependencies > dependencies.txt` gradle doesn't seem to have an output file option for this.
Or if it is public they could also attach a buildscan link.
WDYT?

Reply:
Sounds good.
Maybe for Gradle not the whole `dependencies`, but `gradlew -q dependencies --configuration testRuntimeClasspath` is enough.
No idea whether you can do the same or similar for Maven.